e2e/testcases/private_cert_secret_test.go

```diff
@@ -57,6 +57,10 @@ func secretDataPatch(key, value string) string {
 	return fmt.Sprintf(`{"data": {"%s": "%s"}}`, key, value64)
 }
 
+func secretDataDelete(key string) string {
```

Comment:
Can add comment or `patch` to the name to reduce confusion of the function.

Reply:
changed name to `secretDataDeletePatch`